config/config_cheyenne_gnu.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```

Comment:
One last comment otherwise happy to merge.
Do we need `config_cheyenne_gnu.sh` and `config_cheyenne_intel.sh`
Can we just have one `config_cheyenne.sh`
See `config_hera.sh` for example.

Reply:
I remember we "slacked" about this before. I want to minimize the amount of work and reduce the potential of making mistakes when rolling out a new version of the stack. All it should take is a no-brainer exercise of checking out the code, running the seup_modules script and the build_stack script. No editing of whatsoever files. This should be a failsafe exercise at 3am in the morning when somebody calls you that the libraries got deleted accidentally and they need them back right now. We should do the same for every other platform for which we support multiple compilers, in my opinion. Right now only hera, i.e. `config_hera_gnu.sh` and `config_hera_intel.sh`. And possibly append `_intel` for all the other platforms.

Reply:
sold!

Reply:
Thanks, appreciate it. All CI tests passed, let's wait to hear from @kgerheiser about the jpeg build on NOAA platforms.

Reply:
or maybe not ;-)